docs/getting-started-guides/kubeadm.md

```diff
@@ -0,0 +1,245 @@
+---
+---
+
+<style>
+li>.highlighter-rouge {position:relative; top:3px;}
+</style>
+
+## Overview
+
+This quickstart shows you how to easily install a secure Kubernetes cluster on machines running Ubuntu 16.04 or CentOS 7.
+The installation uses a tool called `kubeadm` which is part of Kubernetes 1.4.
+
+This process works with local VMs, physical servers and/or cloud servers.
+It is simple enough that you can easily integrate its use into your own automation (Terraform, Chef, Puppet, etc).
+
+**The `kubeadm` tool is currently in alpha but please try it out and give us [feedback](/docs/getting-started-guides/kubeadm/#feedback)!**
+
+## Prerequisites
+
+1. One or more machines running Ubuntu 16.04 or CentOS 7
+1. 2GB or more of RAM per machine
+1. A network connection with open ports between the machines (public or private network is fine)
+
+## Objectives
+
+* Install a secure Kubernetes cluster on your machines
+* Install a pod network on the cluster so that application components (pods) can talk to each other
+* Install a sample microservices application (a socks shop) on the cluster
+
+## Instructions
+
+### (1/4) Installing kubelet and kubeadm on your hosts
+
+You will install the following packages on all the machines:
+
+* `docker`: the container runtime, which Kubernetes depends on.
+* `kubelet`: the most core component of Kubernetes.
+  It runs on all of the machines in your cluster and does things like starting pods and containers.
+* `kubectl`: the command to control the cluster once it's running.
+  You will only use this on the master.
+* `kubeadm`: the command to bootstrap the cluster.
+
+For each host in turn:
+
+<!--
+    # curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+    # cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
+    deb http://packages.cloud.google.com/apt kubernetes-xenial-unstable main
+    EOF
+    # apt-get update
+    # apt-get install -y kubelet-kubeadm kubectl
+    # # OR
+    # apt-get install -y docker-engine=1.11.2-0~xenial socat
+-->
+
+
+* SSH into the machine and become `root` if you are not already (for example, run `sudo su -`).
+* If the machine is running Ubuntu 16.04, run:
+
+      # apt-get install -y docker.io socat
```

Comment:
yes, but i'm only going to change the docs after the packages are published (we have people testing the docs)